src/openapi/streaming/connection/transport/websocket-transport.ts

```diff
@@ -649,6 +678,7 @@ class WebsocketTransport implements StreamingTransportInterface {
     ) {
         if (contextId !== this.contextId) {
             this.lastMessageId = null;
+            forceAuth = true;

```

Comment:
I'm not sure this makes a difference. I think it makes sense, but I think if the context id changes then forceAuth will be true.
Except on start, but then I had to add a status guard below so that before start it doesn't request auth.